cmd/gen.go

```diff
@@ -80,19 +82,30 @@ var genCmd = &cobra.Command{
 			}
 		}
 
+		for db, objs := range dbobjs {
+			path := fmt.Sprintf("%s/create_%s.sql", output, db)
+			genType := db + "_script"
+			switch db {
```

Comment:
You'd better only create relevant temporary variables before you need to execute the template.